src/archiefvernietigingscomponent/fixtures/automatic_emails_defaults.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "model": "emails.automaticemail",
+    "pk": 1,
+    "fields": {
+      "type": "review_required",
+      "body": "There is a destruction list ready to be reviewed.",
```

Comment:
Maybe update the default to the examples Jurjen gave? I really want to include all possible variables in each email to show how it works.